<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimeSenseTestClass</class>
 <widget class="QMainWindow" name="PrimeSenseTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>551</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PrimeSenseTest</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1304</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionColor_Over"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionColor_Over">
   <property name="text">
    <string>Color Over</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="primesensetest.qrc"/>
 </resources>
 <connections/>
</ui>
